__tests__/app.test.js

```diff
@@ -187,5 +187,73 @@ describe('App', () => {
         })
       })
     });
+
+    test('should return 400: bad request if passed an article id that is not a number', () => {
+      return request(app)
+      .get('/api/articles/not-an-article/comments')
+      .expect(400)
+      .then((response) => {
+        const body = response.body
+        expect(body.msg).toBe('Bad request')
+      });
+    });
+
+    test('should return 404: not found if passed an article id that is an id not found in the database', () => {
+      return request(app)
+      .get('/api/articles/999999999/comments')
```

Comment:
Be really careful with ids that large, your query will take time and you may error. A solid 99 would do.